app/views/super_admin/app_configs/show.html.erb

```diff
@@ -13,18 +13,25 @@
         <div class="field-unit__label">
           <%= form.label "app_config[#{key}]", @installation_configs[key]&.dig('display_title') || key %>
         </div>
-        <div class="field-unit__field -mt-2 ">
+        <div class="-mt-2 field-unit__field ">
           <% if @installation_configs[key]&.dig('type') == 'boolean' %>
             <%= form.select "app_config[#{key}]", 
               [["True", true], ["False", false]], 
               { selected: ActiveModel::Type::Boolean.new.cast(@app_config[key]) },
               class: "mt-2 border border-slate-100 p-1 rounded-md" 
             %>
+          <% elsif @installation_configs[key]&.dig('type') == 'json' %>
```

Comment:
@sojan-official need your thoughts on this, it re-uses the `locked` attributed, not sure if it is intended for that

Reply:
The locked config is for the page `http://localhost:3000/super_admin/installation_configs`, which we are slowly deprecating. We filter the locked values here and won't display it in the list. 

For this particular case, I think we only need to show this for the chatwoot cloud alone. We don't need to show these values in the UI for even the premium plans. we can add the check based on `DEPLOYMENT_ENV`

Reply:
Makes sense, I guess we can hide it completely. I'll make the required changes